### ISSUE TITLE
Build failure: Waybar

### ISSUE THREAD

**Issue by SignalWalker** [NixOS/nixpkgs#240353]
### Steps To Reproduce

1. `nix build 'github:nixos/nixpkgs/nixpkgs-unstable#waybar'`

### Build log

https://gist.github.com/SignalWalker/dbfa491e464c0d6c951eb90f30191d69

### Notify maintainers

@FlorianFranzen @minijackson @synthetica @lovesegfault @rodrgz

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.9-zen1-1-zen, Arch Linux, noversion, rolling`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.14.1`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

**Comment by fabiandrinksmilk** [comment]
I am getting the same build failure on unstable.

**Comment by alyssais** [comment]
Already fixed in master.  Will be in unstable soon.

**Comment by alyssais** [comment]
(Fix was 0ead7cab3b9ad5ae1e88b792f9b77ba8a605dc5f)

**Comment by NerdHusky** [comment]
Works now, thanks!
